feature/gnoi/system/tests/per_component_reboot_test/metadata.textproto

```diff
@@ -16,3 +16,13 @@ platform_exceptions: {
     gnoi_fabric_component_reboot_unsupported: true
   }
 }
+platform_exceptions: {
+  platform: {
+    vendor: ARISTA
+    hardware_model: "DCS-7280CR3K-32D4"
```

Comment:
Is this deviations needed for other hardware models?  like  "DCS-7808" & "DCS-7804-CH" ?

If not LGTM